src/test/java/com/fasterxml/jackson/core/write/UTF8GeneratorTest.java

```diff
@@ -114,4 +116,37 @@ public void testFilteringWithEscapedChars() throws Exception
         assertNull(p.nextToken());
         p.close();
     }
+
+    public void testHexLowercase() throws Exception
+    {
+        ByteArrayOutputStream bytes = new ByteArrayOutputStream();
+        IOContext ioc = new IOContext(new BufferRecycler(),
+                ContentReference.rawReference(bytes), true);
+        JsonGenerator gen = new UTF8JsonGenerator(ioc, 0, null, bytes, '"');
```

Comment:
Would it be possible to change this and other tests to use `JsonFactory.builder().disable(...)...build()` approach instead? The reason is that this makes merge to `master` (Jackson 3.0) easier: there will no longer be mutability of generators.

Or maybe I can change this after merging PR if you want, either is fine.